.github/workflows/3_LSP_unit_tests_bump.yml

```diff
@@ -0,0 +1,90 @@
+name: Unit tests for LSP and bump LSP version
+
+on:
+  workflow_dispatch:
+    inputs:
+      npm_channel:
+        description: 'Prisma CLI NPM Channel'
+        required: true
+      extension_version:
+        description: 'VSCode version'
+        required: true
```

Comment:
If we are on the master branch yes, so for one step I could instead just use the one from the file system, however, since we also need this version in a different branch, I thought it was less complicated to just pass this as an argument to the next workflow.